### PR TITLE
Fix crawler for MVD asset publishing

### LINKED ISSUE #39: Fix MVD Asset Publishing
## Description

Publishing assets into a dataspace using a contract definition seems to be buggy:
- Sometimes published assets are not visible in other connectors
- Sometimes previously published assets that were visible in the catalog disappear

## Acceptance Criteria
- [ ] Asset publishing within the MVD is tested
- [ ] Fix low hanging fruits
- [ ] Open separate stories for larger bugs

## Tasks

### LINKED ISSUE #167: Fix MVD Asset Publishing
## Description

Publishing assets into a dataspace using a contract definition seems to be buggy:
- Sometimes published assets are not visible in other connectors
- Sometimes previously published assets that were visible in the catalog disappear

## Acceptance Criteria
- [ ] Asset publishing within the MVD is tested
- [ ] Fix low hanging fruits
- [ ] Open separate stories for larger bugs

## Tasks

### PR DESCRIPTION
**Pull request by algattik**
Bump EDC version to include fix https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/1374, which caused federated catalog crawler to crash.

Closes agera-edc/MinimumViableDataspaceFork#39

### CODE REVIEW

**Review by ouphi — APPROVED** [review]

**Review by cpeeyush — APPROVED** [review]

File: launcher/build.gradle.kts [hunk 1]
```diff
@@ -25,12 +25,17 @@ dependencies {
...
-    implementation("${edcGroup}:ids:${edcVersion}")
     implementation("${edcGroup}:observability-api:${edcVersion}")
     implementation("${edcGroup}:data-management-api:${edcVersion}")
     implementation("${edcGroup}:filesystem-configuration:${edcVersion}")
     implementation("${edcGroup}:http:${edcVersion}")
 
+    // IDS
+    implementation("${edcGroup}:ids:${edcVersion}") {
+        // Workaround for https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/1387
+        exclude(group = edcGroup, module = "ids-token-validation")
```
Comment (marcgs) [inline launcher/build.gradle.kts:36]: Just for my understanding, this is unrelated to the bugfix, correct?

**Review by marcgs — APPROVED** [review]
